### PR TITLE
Update ECMA version to accommodate async/await

### PR DESCRIPTION
**Pull request by dhjonesclc**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -2,216 +2,213 @@
...
-            "arrowFunctions": true,           // enable binary literals
-            "blockBindings": true,                     // enable let and const (aka block bindings)
-            "classes": true,                 // enable default function parameters
-            "generators": true,                        // enable generators
-            "objectLiteralComputedProperties": true,   // enable computed object literal property names
-            "objectLiteralShorthandMethods": true,     // enable object literal shorthand methods
-            "objectLiteralShorthandProperties": true,  // enable object literal shorthand properties
-            "templateStrings": true,                   // enable template strings
-            "jsx": false                                // enable JSX
+    'parserOptions': {
```
Comment (dhjonesclc) [inline index.js:5]: Sorry for all the formatting changes. This ecmaVersion line is the only real change.

**Review by zsidnam — APPROVED** [review]
Thanks for doing this!

**Review by pauloliver — APPROVED** [review]
👍

**Comment by cmazakas** [comment]
`co` was better than async/await smh

**Pull request merged**